packages/w3up-client/test/client.test.js

```diff
@@ -528,6 +529,45 @@ export const testClient = {
       )
     },
   }),
+  authorizeGateway: Test.withContext({
+    'should authorize a gateway to serve content from a space': async (
+      assert,
+      { client, mail, grantAccess }
+    ) => {
+      // Step 1: Create a client for Alice and login
+      const aliceEmail = 'alice@web.mail'
+      const aliceLogin = client.login(aliceEmail)
+      const message = await mail.take()
+      assert.deepEqual(message.to, aliceEmail)
+      await grantAccess(message)
+      const aliceAccount = await aliceLogin
+
+      // Step 2: Alice creates a space
+      const spaceA = await client.createSpace('authorize-gateway-space', {
+        account: aliceAccount,
+      })
+      assert.ok(spaceA)
+      await client.setCurrentSpace(spaceA.did())
+
+      // Step 3: Authorize the gateway to serve content from the space
+      const delegation = await client.authorizeGateway(spaceA, {
+        gateway: 'did:web:staging.w3s.link',
+        expiration: Infinity,
+      })
+      assert.ok(delegation)
+
+      // Step 4: Find the delegation for the default gateway
+      assert.equal(delegation.audience.did(), 'did:web:staging.w3s.link')
+      assert.ok(
+        delegation.capabilities.some(
+          // @ts-expect-error
```

Comment:
**suggestion:** Let's not use these, they keep biting us later. What's the error here? It looks like a `Capability => boolean` to me—isn't that what it expects?